quizzes/tests.py

```diff
@@ -0,0 +1,131 @@
+from django.test import TestCase
+
+import datetime
+
+from django.test import TestCase
+from django.utils import timezone
+from django.urls import reverse
+
+from .models import Quiz
+
+# Built following alongside Django Software Foundation's Writing your first Django app Tutorial
+# https://docs.djangoproject.com/en/3.0/intro/tutorial01/
+
+class QuizModelTests(TestCase):
+
+    def test_was_published_recently_with_future_question(self):
```

Comment:
This might be getting into stylistic territory a bit, but I think the function names could be shortened, especially since the descriptive names are fairly redundant with their docstrings.

Reply:
I'm not 100% sure this is possible, but making some instance variables for the QuizModelTests instead of within separate functions might be worthwhile

Reply:
Noted! Made this while following the Django tutorial, so I agree a lot can be changed better fit this module. This section will probably be overhauled soon - I'm considering moving to an active/inactive publishing system so a lot of these could be obsolete in a few PRs.